person.rb

```diff
@@ -1,18 +1,23 @@
 require './nameable'
+require './rental'
 
 class Person < Nameable
-  attr_reader :id
-
   attr_accessor :name, :age
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'unknown', parent_permission: true)
     @id = Random.rand(1...1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
 
+  def add_rental(_rental)
```

Comment:
- [ ] I got an error when I tried to add a rental for a person. This method should take to arguments, `book` & `date`. As of now, your method only takes one argument. And when I pass the one argument, it also throw an error.

See screenshots:

With two arguments passed|
:--------------------------------------:|
![a1](https://user-images.githubusercontent.com/39523312/159056855-e85dc120-f17f-47b8-b5d4-dc4eeb95d9b3.png)

With one argument passed|
:-----------------------------------:|
![a1](https://user-images.githubusercontent.com/39523312/159056972-c7737e61-26b7-468e-b4ce-b02cae0cf0ee.png)